internal/signerverifier/sigstore/sigstore.go

```diff
@@ -154,21 +173,82 @@ func (v *Verifier) ExpectedExtensionKind() string {
 	return ExtensionMimeType
 }
 
-func (v *Verifier) getTUFRoot() (root.TrustedMaterial, error) {
+func (v *Verifier) getTUFRoot() (root.TrustedMaterial, bool, error) {
+	// The env vars we look at
+	// SIGSTORE_ROOT_FILE -> the Fulcio root
+	// SIGSTORE_CT_LOG_PUBLIC_KEY_FILE -> Fulcio's CT Log pubkey
+	// SIGSTORE_REKOR_PUBLIC_KEY -> Rekor's pubkey
+	// TODO: Support tsa
+	// return root.NewTrustedRoot(root.TrustedRootMediaType01, fulcioRoot, ctLog, nil, rekorLog)
+
+	fulcioRootFilePath := os.Getenv("SIGSTORE_ROOT_FILE")
+	ctLogPublicKeyFilePath := os.Getenv("SIGSTORE_CT_LOG_PUBLIC_KEY_FILE")
+	rekorPublicKeyFilePath := os.Getenv("SIGSTORE_REKOR_PUBLIC_KEY")
+
+	if fulcioRootFilePath != "" || ctLogPublicKeyFilePath != "" || rekorPublicKeyFilePath != "" {
```

Comment:
I'm not sure of the right approach here since not all of them may be set? With a private instance that doesn't use TUF, what's our fallback? We sort of need all of these vars?

@wlynch

Reply:
Tracking here: https://github.com/gittuf/gittuf/issues/608